js-conditions-and-loops/conditions.js

```diff
@@ -0,0 +1,40 @@
+//Using "if else"
+const averageGrade = 85;
+
+if (averageGrade <= 59){
```

Comment:
just as advice, use if (averageGrade < 60). It is less confusing. But your code is valid

Reply:
fixed